packages/paste-website/src/pages/core/changing-theme.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Changing the Theme
+description: When using Paste, migrating to a different theme is simple and quick.
```

Comment:
The thing that stands out to me the most is the word "migrating". Which feels a little loaded. We might want to simply change it to swapping or changing or something else.

Reply:
Updated, now it says 'swapping'